embulk-output-sqlserver/src/main/java/org/embulk/output/SQLServerOutputPlugin.java

```diff
@@ -88,6 +88,9 @@
         @ConfigDefault("null")
         public Optional<String> getNativeDriverName();
 
+        @Config("database_encoding")
+        @ConfigDefault("\"MS932\"")
+        public Optional<String> getDatabaseEncoding();
```

Comment:
The default encoding "MS932" is defined twice, here and https://github.com/embulk/embulk-output-jdbc/pull/209/files/407f89822fc10107adbaafec3b350297715cfc59..71a0ac38eda0dfdb3efac69f786f2c6fce9d633d#diff-56b39a385e6ce200b76f90d83899d3baR67 .
This property should not be `Optional<String>` but `String`, because default value is defined.

Reply:
Exactly. This type wideness is unnecessary.